pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -0,0 +1,63 @@
+//go:build linux && rego
+// +build linux,rego
+
+package securitypolicy
+
+import (
+	"context"
+	"errors"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	oci "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const regoEnforcer = "rego"
+
+func init() {
+	registeredEnforcers[regoEnforcer] = CreateRegoEnforcer
+	defaultEnforcer = regoEnforcer
+	log.G(context.Background()).Debugf("registered enforcers: %+v", registeredEnforcers)
+}
+
+type RegoEnforcer struct{}
```

Comment:
can we get a comment to explain what `Rego` is/what policy its enforcing?

Reply:
I'll add a comment about what Rego is.